Sources/App/Models/ToolArgs/QuestionnaireResponseArgs.swift

```diff
@@ -40,6 +40,8 @@ struct QuestionnaireResponseArgs: Codable {
             answer = .text(text)
         } else if let codingWrapper = try? container.decode(CodingWrapper.self, forKey: .answer) {
             answer = .text(codingWrapper.valueCoding.code)
+        } else if try container.decodeNil(forKey: .answer) {
+            answer = .text("null")
```

Comment:
Is there a better way to code this, e.g. something that represents a non-response?

Reply:
made it, so that the json would have an item, but an answer of null (which is most likely not even encoded) to avoid confusion

Reply:
Thank you!